modules/machineset-yaml-azure.adoc

```diff
@@ -71,19 +110,46 @@ spec:
           resourceGroup: <infrastructureID>-rg <1>
           sshPrivateKey: ""
           sshPublicKey: ""
+ifndef::infra[]
           subnet: <infrastructureID>-<role>-subnet <1> <2>
           userDataSecret:
             name: <role>-user-data <2>
+endif::infra[]
+ifdef::infra[]
+          subnet: <infrastructureID>-infra-subnet <1> <2>
+          userDataSecret:
+            name: infra-user-data <2>
+endif::infra[]
           vmSize: Standard_D2s_v3
           vnet: <infrastructureID>-vnet <1>
```

Comment:
the vnet value is not correct, it should be something like qeci-22538-vnet, you can obtain the vnet by running the following command:
```
  $  oc -n openshift-machine-api \
    -o jsonpath='{.spec.template.spec.providerSpec.value.vnet}{"\n"}' \
    get machineset/<infrastructureID>-worker-centralus1
```